aspnetcore/introduction-to-aspnet-core.md

```diff
@@ -4,246 +4,250 @@ author: tdykstra
+:::moniker-end
 
-## Build web APIs and web UI using ASP.NET Core MVC
+:::moniker range=">= aspnetcore-3.0 < aspnetcore-6.0"
 
-ASP.NET Core MVC provides features to build [web APIs](xref:tutorials/first-web-api) and [web apps](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro):
+* [Blazor](xref:blazor/index), a component-based web UI framework based on C# that supports both server-side rendering via the .NET runtime and client-side rendering via WebAssembly.
+* [Razor Pages](xref:razor-pages/index), a page-based programming model that makes building web UI easier and more productive.
+* [Model-View-Controller (MVC)](xref:mvc/overview), a traditional framework for building web apps and web APIs using the [Model-View-Controller design pattern](https://developer.mozilla.org/docs/Glossary/MVC).
 
-* The [Model-View-Controller (MVC) pattern](xref:mvc/overview) helps make your web APIs and web apps testable.
-* [Blazor](xref:blazor/index), a component-based web UI framework based on C# that supports both server-side rendering and client-side rendering via WebAssembly.
-* [Razor Pages](xref:razor-pages/index) is a page-based programming model that makes building web UI easier and more productive.
-* [Razor markup](xref:mvc/views/razor) provides a productive syntax for [Razor Pages](xref:razor-pages/index) and [MVC views](xref:mvc/views/overview).
-* [Tag Helpers](xref:mvc/views/tag-helpers/intro) enable server-side code to participate in creating and rendering HTML elements in Razor files.
-* Built-in support for [multiple data formats and content negotiation](xref:web-api/advanced/formatting) lets your web APIs reach a broad range of clients, including browsers and mobile devices.
-* [Model binding](xref:mvc/models/model-binding) automatically maps data from HTTP requests to action method parameters.
-* [Model validation](xref:mvc/models/validation) automatically performs client-side and server-side validation.
+:::moniker-end
 
-## Client-side development
+:::moniker range="< aspnetcore-3.0"
 
-ASP.NET Core includes [Blazor](xref:blazor/index) for building richly interactive web UI, and also integrates with other popular frontend JavaScript frameworks like [Angular](/visualstudio/javascript/tutorial-asp-net-core-with-angular), [React](/visualstudio/javascript/tutorial-asp-net-core-with-react), [Vue](/visualstudio/javascript/tutorial-asp-net-core-with-vue), and [Bootstrap](https://getbootstrap.com/). For more information, see <xref:blazor/index> and related topics under *Client-side development*.
+* [Razor Pages](xref:razor-pages/index), a page-based programming model that makes building web UI easier and more productive.
+* [Model-View-Controller (MVC)](xref:mvc/overview), a traditional framework for building web apps and web APIs using the [Model-View-Controller design pattern](https://developer.mozilla.org/docs/Glossary/MVC).
 
-<a name="target-framework"></a>
+:::moniker-end
 
 ## ASP.NET Core target frameworks
```

Comment:
This whole section feels outdated to me. It talks about ancient ASP.NET Core versions that are out of support and largely irrelevant. I think we should delete this entire section.